functest/send_money_test.go

```diff
@@ -78,10 +79,10 @@ func TestTransferMoneyToNotExist(t *testing.T) {
 	firstMember := createMember(t, "Member1")
 	oldFirstBalance := getBalanceNoErr(t, firstMember, firstMember.ref)
 
-	amount := 111
+	amount := "10"
 
-	_, err := signedRequest(firstMember, "Transfer", amount, testutils.RandomRef().String())
-	require.Contains(t, err.Error(), "[ Transfer ] Can't get implementation: [ GetDelegate ] on calling main API")
+	_, err := signedRequest(firstMember, "wallet.transfer", map[string]interface{}{"amount": amount, "to": testutils.RandomRef().String()})
+	require.NotNil(t, err)
```

Comment:
Its better to check error message here also, to be sure its about not existed member, not smth else